src/main/java/Task.java

```diff
@@ -0,0 +1,81 @@
+import java.io.Serializable;
+import java.time.format.DateTimeFormatter;
+
+public class Task implements Serializable {
+
+    // task = Full input line
+    // type = todo, event, or deadline
+    // done = done or not done
+
+    String task;
+    String type;
+    boolean isDone = false;
+    static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    static final DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HHmm");
+
+
+    // Tasks represented by their initials
+    private final static String TODO = "T";
+    private final static String DEADLINE = "D";
+    private final static String EVENT = "E";
+
+    public Task(String type, String task) {
+
+        if(type.equals("todo")) {
+            this.type = TODO;
+        } else if(type.equals("deadline")) {
+            this.type = DEADLINE;
+        } else if(type.equals("event")) {
+            this.type = EVENT;
+        }
+
+        this.task = task;
+    }
```

Comment:
I like your use of `final String`s for representing task types.
Do take note of the naming convention of `final` variables, which require a common type prefix (e.g. `TASK_TODO`).
Since you have `Todo`, `Deadline` and `Event` subclasses, consider putting the string constants in their respective subclasses instead.